obsidian.el

```diff
@@ -636,8 +791,22 @@ Opens markdown links in other window if ARG is non-nil.."
     (if (s-contains-p ":" normalized)
         (browse-url normalized)
       (-> normalized
-          obsidian-prepare-file-path
-          (obsidian-find-file arg)))))
+          obsidian--prepare-file-path
+          (obsidian-find-point-in-file 0 arg)))))
+
+(defun obsidian-follow-backlink-at-point ()

```

Comment:
Because we're now including not only the link itself but also the location of the link in the target file, we can jump directly to that location in the target file.